aiohttp==3.9.0b1  breaks pymodbus tests

There is some sort of change in `aiohttp==3.9.0b1` that breaks 2 of the tests.

```python
python3 -m pytest -k simulator_server_tcp .
[...] snip
E       AssertionError: ERROR tasks/threads hanging after 9 retries:
E
E       assert False
```

`aiohttp==3.9.0b0` is fine

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -56,7 +56,7 @@ repl = [
     "click>=8.0.0"
 ]
 simulator = [
-    "aiohttp>=3.9.0b0"
+    "aiohttp==3.9.0b0"
 ]
 documentation = [
     "recommonmark>=0.7.1",

